src/components/pages/jobs/jobs-list/jobs-list.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from 'react';
+
+import Container from 'components/shared/container';
+import Link from 'components/shared/link';
+
+const JobsList = () => {
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
+    fetch(`https://boards-api.greenhouse.io/v1/boards/zenithlabs/jobs?content=true`)
+      .then((response) => response.json())
+      .then(({ jobs }) => {
+        const jobsByDepartmentAsObject = {};
+
+        jobs.forEach(({ title, location, departments, absolute_url }) => {
+          departments.forEach(({ name }) => {
+            if (!jobsByDepartmentAsObject[name]) jobsByDepartmentAsObject[name] = [];
+            jobsByDepartmentAsObject[name].push({
+              name: title,
+              location: location.name,
+              url: absolute_url,
+            });
+          });
+        });
+
+        const jobsByDepartmentAsArray = Object.keys(jobsByDepartmentAsObject).map((department) => ({
+          name: department,
+          items: jobsByDepartmentAsObject[department],
+        }));
+
+        setItems(jobsByDepartmentAsArray);
+      })
+      .catch((error) => console.log(error));
+  }, []);
+
+  return (
+    <section className="safe-paddings pt-64 3xl:pt-60 2xl:pt-52 xl:pt-48 lg:pt-40 md:pt-28">
```

Comment:
I think it is better to keep spacing consistent, thus adjust a bit top padding of the job openings

<img width="464" alt="image" src="https://user-images.githubusercontent.com/32940211/152997756-c37ee0a5-7bd9-4bc1-ad16-c455f006e966.png">

Reply:
I've made it a bit larger but not the same though since it was too much in my opinion

We'll see what designers say on review